### ISSUE TITLE
0.8 pre related question

### ISSUE THREAD

**Issue by brunoamancio** [igraph/igraph#771]
I commented another issue (https://github.com/igraph/igraph/issues/761), but my error is different, so I decided to post a new issue.

I'm trying to compile from source and during the compiling process I get the following error:

gcc -std=gnu99 -I/usr/share/R/include -DNDEBUG -DUSING_R -I. -Iinclude -Ics -Iglpk -Iplfit -ICHOLMOD/Include -IAMD/Include -ICOLAMD/Include -ISuiteSparse_config -g -DNDEBUG -DNPARTITION -DNTIMER -DNCAMD -DNPRINT -DPACKAGE_VERSION=\"@VERSION@\" -DINTERNAL_ARPACK -DIGRAPH_THREAD_LOCAL=/*_/ -fpic -g -c uuid/gen_uuid.c -o uuid/gen_uuid.o
uuid/gen_uuid.c: In function ‘get_clock’:
uuid/gen_uuid.c:362:3: warning: implicit declaration of function ‘flock’ [-Wimplicit-function-declaration]
while (flock(state_fd, LOCK_EX) < 0) {
^
uuid/gen_uuid.c:362:26: error: ‘LOCK_EX’ undeclared (first use in this function)
while (flock(state_fd, LOCK_EX) < 0) {
^
uuid/gen_uuid.c:362:26: note: each undeclared identifier is reported only once for each function it appears in
uuid/gen_uuid.c:434:19: error: ‘LOCK_UN’ undeclared (first use in this function)
flock(state_fd, LOCK_UN);
^
make: *_\* [uuid/gen_uuid.o] Error 1
ERROR: compilation failed for package ‘igraph’

removing ‘/home/th3b0y/R/x86_64-pc-linux-gnu-library/3.1/igraph’ Warning message: In install.packages(path_to_file, repos = NULL, type = "source", : installation of package ‘/home/th3b0y/Downloads/igraph_0.7.999-280.tar.gz’ had non-zero exit status
